tests/bluetooth/tester/src/audio/btp_cap.c

```diff
@@ -22,6 +22,8 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL);
 
 static struct btp_bap_unicast_group *u_group;
 
+static K_SEM_DEFINE(source_stopped_sem, 0U, CONFIG_BT_BAP_BROADCAST_SRC_COUNT);
```

Comment:
Nitpick: Since we only have a single broadcast source in the `btp_cap`, we don't really need to set it above 1.

Reply:
Just seems a bit wrong to hard-code it to 1 :P